include/swift/Basic/TaskQueue.h

```diff
@@ -95,11 +95,12 @@ class TaskQueue {
   /// available and SeparateErrors was true.  (This may not be available on all
   /// platforms.)
   /// \param Context the context which was passed when the task was added
+  /// \param Signal the terminating signal number
```

Comment:
As @jrose-apple pointed out in the linked SR, this should mention that it is not available on all platforms (or that is is specifically available on Unix-likes that use that particular implementation)